lib/dockerize_rails/docker_commands/helpers.rb

```diff
@@ -0,0 +1,27 @@
+module DockerizeRails
+  module DockerCommands
+    module Helpers
+      NAMES = {
+        image: {
+            rails: "#{DRConfig.application_name}_rails:#{DRConfig.application_env}".freeze,
+            mysql: "#{DRConfig.application_name}_mysql:#{DRConfig.application_env}".freeze,
+            postgres: "#{DRConfig.application_name}_postgres:#{DRConfig.application_env}".freeze,
+        }.freeze,
+        container: {
+            rails: "#{DRConfig.application_name}_rails_container".freeze,
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

<!-- Comment by [SideCI](https://sideci.com) -->